packages/jest-circus/src/__tests__/hooks_error.test.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const circus = require('../index.js');
+
+describe('hooks error throwing', () => {
+  test.each([['beforeEach'], ['beforeAll'], ['afterEach'], ['afterAll']])(
```

Comment:
we don't need the nested arrays, can be just `test.each(['beforeEach', 'beforeAll', 'afterEach', 'afterAll'])(`